### PR TITLE
[Not ticket] - Fix for [PROGRAMMES-6274]

### PR DESCRIPTION
**Pull request by tatitati**
Original PR:
https://github.com/bbc/programmes-frontend/pull/555/files

RmsPodcast might be null, and is defined as such but I forgot it in one place. I will try to create a good end to end to ensure that this doesn't explode anymore for this special case.

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/DetailsPresenter.php [hunk 1]
```diff
@@ -137,7 +137,11 @@ public function getPodcastFileName(): string
 
     public function isUkOnlyPodcast(): bool
     {
-        return $this->rmsPodcast->isOnlyInUk();
+        if ($this->rmsPodcast) {
```
Comment (votemike) [inline src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/DetailsPresenter.php]: In the file docblocks, it says it can't be null...

Comment (tatitati) [reply src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/DetailsPresenter.php]: Done

File: src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/DetailsPresenter.php [hunk 2]
```diff
@@ -28,7 +28,7 @@ class DetailsPresenter extends Presenter
     /** @var UrlGeneratorInterface */
     private $router;
 
-    /** @var RmsPodcast */
+    /** @var RmsPodcast|null */
```
Comment (benolot) [inline src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/DetailsPresenter.php]: So if it's not a 200, it passed null into the presenter right?

Comment (tatitati) [reply src/Ds2013/Presenters/Section/Episode/Map/Panels/Main/DetailsPresenter.php]: Yes, as you can see here:
https://github.com/bbc/programmes-frontend/blob/develop/src/ExternalApi/RmsPodcast/Service/RmsPodcastService.php#L54

**Pull request merged**